From 2510373ed5e4907458fd5740fa91f24d70b79d2f Mon Sep 17 00:00:00 2001
From: Beverly Tai <beverlyt@google.com>
Date: Thu, 14 Sep 2023 20:51:27 +0000
Subject: [PATCH] Revert "On device lockdown, always show the keyguard"

This reverts commit 100ae42365d7fc8ba7d241e8c9a7ef6aa0cdb961.

Reason for revert: b/300463732 regression
Bug: 300463732
Bug: 218495634
(cherry picked from https://googleplex-android-review.googlesource.com/q/commit:996896e672f28aa96a3d8158192de3cd4a105bc3)
Merged-In: Ida8f2ab5e4ac79b38c21ba7bacfe9c374c9cc600
Change-Id: Ida8f2ab5e4ac79b38c21ba7bacfe9c374c9cc600
---
 .../systemui/keyguard/KeyguardViewMediator.java        | 10 +---------
 1 file changed, 1 insertion(+), 9 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java b/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
index 5d086e833e14..60137804a066 100644
--- a/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
+++ b/packages/SystemUI/src/com/android/systemui/keyguard/KeyguardViewMediator.java
@@ -702,13 +702,6 @@ public class KeyguardViewMediator extends CoreStartable implements Dumpable,
                 }
             }
         }
-
-        @Override
-        public void onStrongAuthStateChanged(int userId) {
-            if (mLockPatternUtils.isUserInLockdown(KeyguardUpdateMonitor.getCurrentUser())) {
-                doKeyguardLocked(null);
-            }
-        }
     };
 
     ViewMediatorCallback mViewMediatorCallback = new ViewMediatorCallback() {
@@ -1783,8 +1776,7 @@ public class KeyguardViewMediator extends CoreStartable implements Dumpable,
         }
 
         // if another app is disabling us, don't show
-        if (!mExternallyEnabled
-            && !mLockPatternUtils.isUserInLockdown(KeyguardUpdateMonitor.getCurrentUser())) {
+        if (!mExternallyEnabled) {
             if (DEBUG) Log.d(TAG, "doKeyguard: not showing because externally disabled");
 
             mNeedToReshowWhenReenabled = true;
-- 
2.42.0.820.g83a721a137-goog

